File: algorithms(python)/bubble_sort.py
from typing import List

def bubbleSort(numbers: List):
    for i in range(0, len(numbers)-1):
        for j in range(0, len(numbers)-i-1):
            if numbers[j] > numbers[j+1]:
                numbers[j], numbers[j+1] = numbers[j+1], numbers[j]
    

File: algorithms(dart)/bubbleSort.dart
void bubbleSort(List<int> numbers) {
  // sort algorithm for bubble sort.
  int temp;
  for (int i = 0; i < numbers.length - 1; i++) {
    for (int j = 0; j < numbers.length - i - 1; j++) {
      if (numbers[j] > numbers[j + 1]) {
        temp = numbers[j];
        numbers[j] = numbers[j + 1];
        numbers[j + 1] = temp;
      }
    }
  }
}

